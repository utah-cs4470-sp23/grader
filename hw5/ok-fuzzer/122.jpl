print "|%?]@]=#_$"
read image "=^?)" to A
type f2._ = x
show true
time write image { A ( ) , [ cC ] , {} , ( false ) , G ( j ( false , false ) ) } to ""
assert {}, ""
show [ t_ ]
read image "'^" to F_614n241_p10L
let {} = [ {} ]
write image ! false to ""
show ( s_ )
