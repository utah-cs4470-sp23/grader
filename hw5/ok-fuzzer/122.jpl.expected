(PrintCmd "|%?]@]=#_$")
(ReadCmd "=^?)" (VarArgument A))
(TypeCmd f2._ (VarType x))
(ShowCmd (TrueExpr))
(TimeCmd
 (WriteCmd
  (TupleLiteralExpr
   (CallExpr A)
   (ArrayLiteralExpr (VarExpr cC))
   (TupleLiteralExpr)
   (FalseExpr)
   (CallExpr G (CallExpr j (FalseExpr) (FalseExpr))))
  ""))
(AssertCmd (TupleLiteralExpr) "")
(ShowCmd (ArrayLiteralExpr (VarExpr t_)))
(ReadCmd "'^" (VarArgument F_614n241_p10L))
(LetCmd (TupleLValue) (ArrayLiteralExpr (TupleLiteralExpr)))
(WriteCmd (UnopExpr ! (FalseExpr)) "")
(ShowCmd (VarExpr s_))
